src/Microsoft.ServiceHub.Framework/Extensions/ServiceManagerReflectionHelpers.cs

```diff
@@ -0,0 +1,169 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics;
+using Microsoft.ServiceHub.Framework.Services;
+using Nerdbank.Streams;
+
+namespace Microsoft.ServiceHub.Framework.Extensions
+{
+#pragma warning disable CS1574 // XML comment has cref attribute that could not be resolved
+	/// <summary>
+	/// Extension methods used via reflection in IServiceManager.
+	/// <seealso cref="ServiceHub.HostStub.IServiceManager"/>
```

Comment:
Can we drop these non-resolvable xml links instead of suppress the warnings? They won't ever link again, even within DevCore use I think, so just typing out the type name is just as good, without the warnings+suppressions.

Reply:
Done.